wasm/examples/parcel_example/index.js

```diff
@@ -62,6 +74,63 @@ async function renderRiveAnimation({ rive, num, hasRandomSizes }) {
   // Don't use the offscreen renderer for FF as it should have a context limit of 300
   const renderer = rive.makeRenderer(canvas, true);
 
+  const activeStateMachines = stateMachine ? [stateMachine] : [];
+  function mouseCallback(event) {
```

Comment:
This looks like duplicated code. Is it possible to somehow share with rive.ts?

Reply:
Yup, moved to a util - in the latest commit!